.github/workflows/distro.yml

```diff
@@ -0,0 +1,324 @@
+name: Distro
+
+on:
+  workflow_dispatch:
+    inputs:
+      DEBUG_ENABLED:
+        description: 'Run the build with tmate debugging enabled (https://github.com/marketplace/actions/debugging-with-tmate)'
+        type: boolean
+        required: false
+        default: false
+      DEBUG_OS:
+        description: 'which runner os to run the tmate action in (if the tmate action is run)'
+        type: string
+        default: 'windows-2019'
+        required: false
+      DEBUG_ARCH:
+        description: 'which runner arch to run the tmate action in (if the tmate action is run)'
+        type: string
+        default: 'x86_64'
+        required: false
+      DEBUG_DETACHED:
+        description: 'whether to launch tmate in detached mode (if the tmate action is run)'
+        type: boolean
+        required: false
+        default: true
+      AIE_COMMIT:
+        description: 'which aie commit to build'
+        type: string
+        required: false
+        default: ''
+
+  pull_request:
+  release:
+    types:
+      - published
+
+jobs:
+
+  set_datetime:
+
+    name: Set canonical datetime
+
+    runs-on: ubuntu-latest
+
+    outputs:
+      DATETIME: ${{ steps.set_datetime.outputs.DATETIME }}
+
+    steps:
+      - name: set datetime
+        id: set_datetime
+        shell: bash
+        run: |
+          DATETIME=$(date +"%Y%m%d%H")
+          echo "DATETIME=${DATETIME}" | tee -a $GITHUB_OUTPUT
+
+  build_wheels:
+
+    needs: [set_datetime]
+
+    name: ${{ matrix.OS }} ${{ matrix.ARCH }}
+
+    continue-on-error: true
+
+    runs-on: ${{ matrix.OS }}
+
+    outputs:
+      MLIR_AIE_WHEEL_VERSION: ${{ steps.get_wheel_version.outputs.MLIR_AIE_WHEEL_VERSION }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - OS: ubuntu-20.04
+            ARCH: x86_64
+
+          - OS: ubuntu-20.04
+            ARCH: aarch64
+
+          - OS: macos-11
+            ARCH: x86_64
+
+          - OS: macos-11
+            ARCH: arm64
+
+          - OS: windows-2019
+            ARCH: AMD64
+
+    steps:
+    # Enable tmate debugging of manually-triggered workflows if the input option was provided
+    - name: Setup tmate session
+      uses: mxschmitt/action-tmate@v3
+      if: ${{ inputs.DEBUG_ENABLED && inputs.DEBUG_OS == matrix.OS && inputs.DEBUG_ARCH == matrix.ARCH }}
+      with:
+        limit-access-to-actor: true
+        detached: ${{ inputs.DEBUG_DETACHED }}
+
+    - uses: actions/checkout@v3
+
+    - uses: actions/checkout@v3
```

Comment:
You should add a `name` to help the reading of the output to avoid confusion with previous step.